config.yaml

```diff
@@ -18,6 +18,13 @@ options:
     type: string
     default: 192.168.250.1
     description: Gateway IP address to the Core Network.
+  core-interface-mtu-size:
+    type: int
+    description: |
+      MTU size of UPF's Core Interface which is allowed
+      between 1200 < MTU <= 65536 (bytes).
+      If config is not provided, Multus creates interface
+      with default MTU size which is 1500 bytes.
```

Comment:
I am a little confused. I don't see this part being implemented in the code. Is it done by Multus so it is out of the scope of this charm ?
should we set a default value = 1500 to the core-inteface-mtu-size config instead of handling the <None> case in the config? 

Same comment for access-inteface-mtu-size

Reply:
Default MTU size is set by Multus. We do not need to do something to set default MTU. We are explaining here that if user does not configure it, interface comes with MTU size 1500.